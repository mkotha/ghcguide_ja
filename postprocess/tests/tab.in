<html>
<body>
<p>
	text
</p>
</body>
</html>
