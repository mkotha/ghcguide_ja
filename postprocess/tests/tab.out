
<html lang="en" xml:lang="en" xmlns="http://www.w3.org/1999/xhtml">
  <body>
    <p>text</p>
  </body>
</html>
